bin/cron/deploy_to_levelbuilder

```diff
@@ -65,7 +65,7 @@ def new_commits?(commit)
 end
 
 def message_dtl_soon(pr_number)
-  'robo-DTL will deploy <a href="#{GitHub.url(pr_number)}">PR#{pr_number}</a> in five ' \
+  "<!here> Robo-DTL will deploy <a href=\"#{GitHub.url(pr_number)}\">PR #{pr_number}</a> in five " \
```

Comment:
[Here's the documentation](https://api.slack.com/docs/message-formatting#variables) on the `<!here>` syntax for triggering mentions from app integrations.